app/src/main/java/tech/pauly/findapet/utils/BindingAdapters.java

```diff
@@ -15,4 +32,45 @@ public static void navigationItemSelectedListener(BottomNavigationView bottomNav
     public static void defaultSelectedItem(BottomNavigationView bottomNavigationView, @IdRes int selectedItemResId) {
         bottomNavigationView.setSelectedItemId(selectedItemResId);
     }
+
+    @BindingAdapter("imageUrl")
+    public static void loadImageIntoView(ImageView view, String url) {
+        if (!TextUtils.isEmpty(url)) {
+            final Transformation transformation = new RoundedCornersTransformation(10, 0);
+            Picasso.with(view.getContext())
+                   .load(Uri.parse(url))
```

Comment:
Just realized Picasso is still hitting real internet so the tests definitely aren't idempotent under network changes. Probably just need to make a non static Picasso Provider that is mocked for Espresso.

Reply:
Not going to do this until I need to. Right now Picasso just silently fails and all my tests are still fine. I open up a can of worms trying to turn this non-static since it's in BindingAdapters. Until theres a use case, just going to ignore it.